src/auth/token.vue

```diff
@@ -61,13 +61,17 @@ export default {
 	},
 	created: function() {
 		if (this.$route.query.missingcsc) {
-			this.$router.replace({name: 'home', params: {missingCsc: true}})
+			this.$auth.setLoginError("missingcsc")
+			this.$auth.setUser(null)
+			this.$router.replace({name: 'home'})
 			return
 		}
-		
+
 		// User should have home organization
 		if (this.$route.query.missingorg) {
-			this.$router.replace({name: 'home', params: {missingOrg: true}})
+			this.$auth.setLoginError("missingorg")
+			this.$auth.setUser(null)
+			this.$router.replace({name: 'home'})
```

Comment:
maybe this router call should be also, maybe this would be inside this.$auth.setLoginError or some other wrapper